curriculum/js1/8.js

```diff
@@ -1,12 +1,15 @@
 /**
- * Greatest Common Denominator - returns greatest common denominator
+ * Greatest Common Divisor - returns greatest common divisor 
  * @param {number} a
  * @param {number} b
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, counter = 1, gcd) => {
+  if (a < counter || b < counter) return gcd;
+  if (a === b) return a;
```

Comment:
this is a redundant check. If a === b, then they would both be divisible by counter, which your next if condition would check.

Reply:
you righ